src/Valet/Services/DockerService.cs

```diff
@@ -120,10 +119,10 @@ private IEnumerable<string> GetEnvironmentVariableArguments()
             if (string.IsNullOrWhiteSpace(value)) continue;
 
             var key = env;
-            if (key.StartsWith("GH_"))
-                key = key.Replace("GH_", "GITHUB_");
+            if (key.StartsWith("GH_", StringComparison.Ordinal))
```

Comment:
This ensures that the env var string comparison is case in-sensitive correct?

Reply:
Correct!